cypress/e2e/planForm.cy.js

```diff
@@ -0,0 +1,97 @@
+describe("Init View", () => {
+    beforeEach(() => {
+        cy.visit("http://localhost:5173/malker");
+    });
+
+    it('Component "Header" check', () => {
+        cy.get('div[id="header"]').should("exist");
+    });
+
+    it('Component "Footer" check', () => {
+        cy.get('footer[id="footer"]').should("exist");
+    });
+
+    it("Screen components check", () => {
+        cy.get('h1[id="header"]').should("exist");
+        cy.get('p[id="subheader"]').should("exist");
+
+        cy.get('input[id="title"]').should("exist");
+        cy.get('input[id="author"]').should("exist");
+        cy.get('input[id="date"]').should("exist");
+        cy.get('input[id="location"]').should("exist");
+        cy.get('textarea[id="description"]').should("exist");
+
+        cy.get('button[id="Clear"]').should("exist");
+        cy.get('button[id="Create"]').should("exist");
+    });
+});
+
+describe("Functionality Tests", () => {
+    beforeEach(() => {
+        cy.visit("http://localhost:5173/malker");
+    });
+
+    it('Fill in the fields and delete them with the "Clear" button', () => {
```

Comment:
Use shorter names for the tests, please. For example, in this one I would put something like "Check form cleanup".